docs/source/installation.rst

```diff
@@ -81,11 +81,11 @@ After that, you can install the package in development mode
 
     pip install -e ".[dev]"
 
-To compile the Java classes (at the beginning or after changes), run
+To compile the Rust code (at the beginning or after changes), run
```

Comment:
To compile the Rust code (after changes), run.

The beginning part isn't needed since pip install handles the rust part as well